crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -526,8 +505,19 @@ impl ArtifactFetcher {
             return Ok(file);
         }
 
-        // TODO:
         // Otherwise, fall back to scraping from the Web.
+        if let Some(url) = key.abs_path() {
```

Comment:
This should be guarded with the `allow_scraping` option that we pass as the part of request payload from the monolith.

Reply:
```rust
pub struct JsSymbolicationRequestBody {
    #[serde(default)]
    pub source: Option<SentrySourceConfig>,
    #[serde(default)]
    pub stacktraces: Vec<JsStacktrace>,
    #[serde(default)]
    pub modules: Vec<RawObjectInfo>,
    #[serde(default)]
    pub dist: Option<String>,
}
```

This should be extended with:

```rs
#[serde(default = true)]
pub allow_scraping: bool
```

(default in monolith is `True`)